GTViewController/Classes/Custom Classes/GTInputFieldView.h

```diff
@@ -0,0 +1,15 @@
+//
+//  GTInputField.h
+//  GTViewController
+//
+//  Created by Ryan Carlson on 4/13/16.
+//  Copyright © 2016 Michael Harrison. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface GTInputFieldView : UIView<UITextFieldDelegate>
+
+- (instancetype)createInputFieldFromElement:(TBXMLElement *)element withY:(CGFloat)yPos withStyle:(GTPageStyle*)style presentingView:(UIView *)presentingView;
```

Comment:
generally you would miss the create off of a creation method in objective c.
`- (instancetype)inputFieldFromElement:(TBXMLElement *)element withY:(CGFloat)yPos withStyle:(GTPageStyle*)style presentingView:(UIView *)presentingView;`